packages/client/src/utils/test-ids.ts

```diff
@@ -47,7 +47,7 @@ export const OUTCOME_SECTION = "outcome-section";
 export const OUTCOME_SECTION_CONTENT = "outcome-section-content";
 export const NEED_CONTACT = "need-contact";
 export const NEED_PERMIT = "need-permit";
-export const NEED_PERMIT_BUTTON = "need-permit-button";
+export const OUTCOME_MAIN_CONTENT_BUTTON = "outcome-main-content-button";
```

Comment:
In the file `packages/e2e/selectors.js` there is a selector that is still called `[data-testid=need-permit-button]`.